polkadot/node/core/backing/src/tests/mod.rs

```diff
@@ -2256,3 +2269,51 @@ fn new_leaf_view_doesnt_clobber_old() {
 		virtual_overseer
 	});
 }
+
+// Test that a disabled local validator doesn't do any work on `CandidateBackingMessage::Second`
+#[test]
+fn disabled_validator_doesnt_distribute_statement() {
```

Comment:
note that validators start work either by receiving a collation from a validator (seconding) or by seeing another validator second the candidate.

`fn kick_off_validation_work` is probably where the gate needs to be, and both should be tested.

Reply:
Also fixed.